# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xfree86/ftconfig-h-hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- xc/lib/font/FreeType/ftconfig.h.orig	2003-11-02 05:43:24.000000000 +0100
+++ xc/lib/font/FreeType/ftconfig.h	2003-11-06 20:22:06.000000000 +0100
@@ -63,7 +63,7 @@
 
   /* The number of bytes in an `int' type.  */
 
-#include "Xmd.h"
+/* #include <X11/Xmd.h> */
 
 #define FT_SIZEOF_INT 4
 #ifdef LONG64
